axelar-chains-config/info/testnet.json

```diff
@@ -1010,6 +1011,9 @@
       "gasOptions": {
         "gasLimit": 300000000
       },
+      "onchainGasEstimate": {
+        "chainName": "filecoin"
```

Comment:
use `axelarId` field for the chain name instead of adding another one

Reply:
But `axelarId` is `filecoin-2`, so wanted to have a canonical name there too.

Reply:
that refers to a different testnet. The gateway requires apps to use the axelar registered Id otherwise it'll not be recognized, they should pass the same value to the gas service

Reply:
gotcha, will remove that